packages/opentelemetry-core/src/common/types.ts

```diff
@@ -22,6 +22,12 @@ export enum LogLevel {
   DEBUG,
 }
 
+/**
+ * This is equivalent to:
+ * type LogLevelString = 'ERROR' | 'WARN' | 'INFO' | 'DEBUG';
+ */
+export type LogLevelString = keyof typeof LogLevel;
```

Comment:
Very cool you can do this!

Reply:
Yeah, I was really happy to stumble upon it here  https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time 😄